karma.config.cjs

```diff
@@ -46,19 +46,29 @@ module.exports = function (config) {
     // Available frameworks: https://npmjs.org/browse/keyword/karma-adapter
     frameworks: ['jasmine', 'karma-puppeteer-client'],
 
-    // Base path that will be used to resolve all patterns (eg. files, exclude).
-    basePath: 'assets/src/edit-story',
-
     // list of files / patterns to load in the browser
-    files: [{ pattern: '**/karma/**/*.js', watched: false }],
+    files: [
+      { pattern: 'assets/src/edit-story/**/karma/**/*.js', watched: false },
+      {
+        pattern: '__static__/**/*',
+        watched: false,
+        included: false,
+        served: true,
+        nocache: false,
+      },
+    ],
 
     // list of files / patterns to exclude
     exclude: ['**/test/**/*.js'],
 
     // preprocess matching files before serving them to the browser
     // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
     preprocessors: {
-      '**/karma/**/*.js': ['webpack', 'sourcemap'],
+      'assets/src/edit-story/**/karma/**/*.js': ['webpack', 'sourcemap'],
```

Comment:
Let's also include the dashboard here already? (see e.g. #1998)

Reply:
Hmm. Interesting. TBH I did 0 work to enable Dashboard here or try anything relevant to it, tbh.  I think we need to discuss it first. The time will increase and not everything is clear to me, tbh.